src/api/uploads/UploadReachability.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @flow
+ * @file Helper for Box uploads reachability test
+ * @author Box
+ */
+
+import Base from '../Base';
+import LocalStore from '../../util/LocalStore';
+import { DEFAULT_HOSTNAME_UPLOAD } from '../../constants';
+import '../../util/regenerator-runtime';
```

Comment:
there might be better way than import regenerator-runtime every time using async in a file